python/cugraph/core/k_core.py

```diff
@@ -0,0 +1,65 @@
+# Copyright (c) 2019, NVIDIA CORPORATION.
```

Comment:
would recommend renaming the directory here as well as `python/cugraph/core` can mean something else.